setup.py

```diff
@@ -20,5 +35,8 @@
     url='https://github.com/DBuildService/dockerfile-parse',
     license="BSD",
     packages=find_packages(exclude=["tests"]),
+    install_requires=_install_requirements(),
+    setup_requires=['pytest-runner'],
```

Comment:
Keep in mind that this will fail RPM builds, this line must be nuked in spec file. Not sure about `tests_require`.